onnx/defs/nn/defs.cc

```diff
@@ -1024,7 +1086,11 @@ ONNX_OPERATOR_SET_SCHEMA(
             "in effect, the operation expects input data tensor "
             "to arrive with the dimension denotation of [DATA_BATCH, "
             "DATA_CHANNEL, DATA_FEATURE, DATA_FEATURE ...].",
-            "T1")
+            "T1",
+            OpSchema::Single,
+            true,
+            1,
+            OpSchema::Differentiable)
```

Comment:
This one I am not sure. Conv itself is differentiable, but its type conflicts with my mental assumption --- integer tensors are not differentiable.